cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/export/CgmesExportContext.java

```diff
@@ -111,7 +111,7 @@ public static final class ModelDescription {
         private int version = 1;
         private final List<String> dependencies = new ArrayList<>();
         private String modelingAuthoritySet = "powsybl.org";
-        private String id = null;
+        private Set<String> ids = new HashSet<>();
```

Comment:
I assume you need multiple ids for merged models. True?

Having multiple ids inside a model description is confusing for me. I think we should have multiple model descriptions instead. What do you think?

Reply:
Currently the unit tests do not have any situation where we have more than one identifier in the model description

Reply:
Yes, it is for CGM! In this case, since there is several EQ files for example, we will have several EQ ids. It is the case in the test in `StateVariableExportTest.microGridAssembled()`. If you print the export file, you have:
```java
<iidm:network xmlns:iidm="http://www.powsybl.org/schema/iidm/1_8" id="urn:uuid:77b55f87-fc1e-4046-9599-6c6b4f991a86" caseDate="2014-06-01T10:30:00.000Z" forecastDistance="0" sourceFormat="CGMES" minimumValidationLevel="STEADY_STATE_HYPOTHESIS">
    <iidm:property name="CGMES.EQ_BD_ID" value="urn:uuid:2399cbd0-9a39-11e0-aa80-0800200c9a66"/>
    <iidm:property name="CGMES.EQ_ID" value="urn:uuid:d400c631-75a0-4c30-8aed-832b0d282e73"/>
    <iidm:property name="CGMES.SSH_ID" value="urn:uuid:52b712d1-f3b0-4a59-9191-79f2fb1e4c4e"/>
    <iidm:property name="CGMES.SSH_ID#0" value="urn:uuid:66085ffe-dddf-4fc8-805c-2c7aa2097b90"/>
    <iidm:property name="CGMES.TP_BD_ID" value="urn:uuid:2399cbd1-9a39-11e0-aa80-0800200c9a66"/>
    <iidm:property name="CGMES.TP_ID" value="urn:uuid:5d32d257-1646-4906-a1f6-4d7ce3f91569"/>
    <iidm:property name="CGMES.TP_ID#0" value="urn:uuid:f2f43818-09c8-4252-9611-7af80c398d20"/>
```
so several TP and SSH files.

Reply:
Ok. After your clarifications it is ok for me. As we discussed, we just need to be aware that our current implementation of `ModelDescription` is not intended to correspond to the CGMES `ModelDescription` class.